RELEASE_NOTES.rst

```diff
@@ -1,3 +1,121 @@
+       .yhho`          shhho``ohhhs           ohhy.   
+       .ss+           hhhhy   -yhhhh           /so.   
+                    -yhhhs      shhhy-                
+           `///////+yhhh+        /hhhh+/////+/`       
+           /hhhhhhhhhhho          +hhhhhhhhhhh+       
+            `..--ohhhhhy/        :yhhhhhs--.``        
+                :hhhoohhh+      +hhhoohhh:            
+                -oo:  :yhho    ohhy:  :oo-            
+                       `+o-    -++`
+
+For this release we had 168 commits contributed by 66 unique contributors
+over 2.5 months
+
+The name of this release is PolarBear to reflect changes we have made to our
+release process. We have implemented a release freeze before all releases to
+to serve as a testing period for all staged changes. This should also help
+us keep releases on schedule.
+
+This release was an enhancement release which brings the usual slew of
+enhancements to documentation and api. We have started preparing the codebase
+for the big changes that are outlined in the cEPs like section inheritance
+and the next generation core.
+
+**General:**
+
+- Test cases have been improved across the board so they catch more errors
+  before production.
+- Log output via json has been added which is accessible by using the
+  ``--log-json`` option
+- coala domain has been changed from ``coala-analyser.org`` to ``coala.io``.
+  Please file a bug if you find any broken links or instances of the old url.
+
+**Usability:**
+
+- Ignore statements in multi-line comments should be supported properly now
+- emacs has been added to list of editors that coala supports. coala will
+  now output a warning if you specify an editor that has no configuration
+- The ``noqa`` keyword used by other linters as an ignore keyword is now
+  supported as a coala ignore keyword
+- Bears will now check for prerequisites using specified requirements
+  before running. Developers, either override ``check_prequisites`` in
+  your bear or write a new requirement class if you wish to have a custom
+  check.
+- Default coafile warning informs users how to suppress it
+- ``libclang`` update to 3.2 is known to cause problems from some users. Please
+  uninstall and reinstall it if coala tells you there is a version mismatch
+  in ``libclang``
+- ``C#`` now has proper language support and should work with AnnotationBear
+
+**Deprecations:**
+
+- ``change_line`` method in ``Diff.py`` has been deprecated and has been
+  scheduled for removal
+- ``format_str`` setting has been completely removed. Use ``format`` instead
+  to specify a formatting string
+
+**Developers:**
+
+- Travis has been enabled for developers to test their changes on their own CI
+- Changes have been made to developer docs to (hopefully) make contributing
+  easier
+- API docs are now part of the coala repo
+- Unknown Languages are now allowed
+- ``@linter`` decorator now warns when using unsupported or anonymous capture
+  groups
```

Comment:
You should also say "Please adapt regexes appropriately to avoid unnecessary warnings." or so^^